### PR TITLE
Feature/tds config docker

### PR DESCRIPTION
**Pull request by jeffjohnson9046**
Docker support:
* **Dockerfile:**  create the Docker image
* **docker-compose.yml:**  easily stand up the Docker container from the built image
* **docker-startup.sh:**  launch the config service jar w/specified properties

Added instructions to README for interacting with Docker container.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,103 @@ The `TDS_ConfigService` (aka Config Support Service) consists of two modules:
...
+
+* `mvn clean install -f /path/to/service/pom.xml`
+
+To build the service and run integration tests:
+  
+* `mvn clean install -Dintegration-tests.skip=false -f /path/to/service/pom.xml`
+
+To build the service and its associated Docker image:
+
+* `mvn clean install docker:build -f /path/to/service/pom.xml`
```
Comment (jtreuting) [inline README.md]: Should we add any info on where to download and install what is needed for Docker?

File: service/pom.xml [hunk 2]
```diff
@@ -160,12 +164,12 @@
...
                     <execution>
                         <id>flyway_test_migrations</id>
-                        <phase>compile</phase>
+                        <phase>test</phase>
                         <goals>
                             <goal>migrate</goal>
                         </goals>
                         <configuration>
-                            <!-- no point in running migrations if integration tests are skipped -->
+                            <!-- do not migrations if integration tests are skipped -->
```
Comment (jtreuting) [inline service/pom.xml]: "do not run migrations"

**Comment by jtreuting** [comment]
Couple small comments, but looks good to me.

**Comment by millerts** [comment]
lgtm

**Pull request merged**